routes/things/create.ts

```diff
@@ -3,18 +3,76 @@ import { z } from "zod"
 
 export default withRouteSpec({
   methods: ["POST"],
-  jsonBody: z.object({
-    name: z.string(),
-    description: z.string(),
-  }),
+  jsonBody: z
```

Comment:
use winterspec convention and remove all the header stuff. it's on `req.formData`

```suggestion
  formData: z
```

Reply:
use this

![image](https://github.com/user-attachments/assets/f40f9bcd-0072-4685-8083-e16e8b2ce7b7)